docs/source/guide/script_examples.md

```diff
@@ -0,0 +1,448 @@
+ });
+
+
+ LSI.on('entityCreate', region => {
+   if (!isShiftKeyPressed) return; 
+
+   if (window.BULK_REGIONS) return;
+   window.BULK_REGIONS = true;
+   setTimeout(() => window.BULK_REGIONS = false, 1000);
+
+   const existingEntities = Htx.annotationStore.selected.regions;
+
+   setTimeout(() => {
+     // Prevent tagging a single character
+     if (region.text.length < 2) return;
+     regexp = new RegExp(region.text.replace("\\\\n", "\\\\s+").replace(" ", "\\\\s+"), "gi")
+     const matches = Array.from(region.object._value.matchAll(regexp));
+     matches.forEach(m => {
+       if (m.index === region.startOffset) return;
+
+       const startOffset = m.index;
+       const endOffset = m.index + region.text.length;
+
+       // Check for existing entities with overlapping start and end offset
+       let isDuplicate = false;
+       for (const entity of existingEntities) {
+         if (startOffset <= entity.globalOffsets.end && entity.globalOffsets.start <= endOffset) {
+           isDuplicate = true;
+           break;
+         }
+       }
+
+       if (!isDuplicate) {
+         Htx.annotationStore.selected.createResult({
+             text: region.text,
+             start: "/span[1]/text()[1]",
+             startOffset: startOffset,
+             end: "/span[1]/text()[1]",
+             endOffset: endOffset
+           }, {
+             labels: [...region.labeling.value.labels]
+           },
+           region.labeling.from_name,
+           region.object
+         );
+       }
+     });
+
+     Htx.annotationStore.selected.updateObjects();
+   }, 100);
+ });
+```
+
+**Related LSI instance methods:**
+
+* [on(eventName, handler)](scripts#on-eventName-handler)
+
+**Related frontend APIs:**
+
+* [regions](scripts#regions)
+  
+**Related frontend events:**
+
+* [entityCreate](frontend_reference#entityCreate)
+* [entityDelete](frontend_reference#entityDelete)
+
+
+
+#### Labeling config
+
+```xml
+<View>
+  <Header>
+    Labels
+  </Header>
+ <View style="padding: 0em 1em; background: #f1f1f1; margin-right: 1em; border-radius: 3px">
+  <View style="position: sticky; top: 0; height: 50px; overflow: auto;">
+    <Labels name="label" toName="text">
+      <Label value="type_1" background="#3a1381"/>
+      <Label value="type_2" background="#FFA39E"/>
+      <Label value="type_3" background="#46ae19"/>
+      <Label value="type_4" background="#8ab1c1"/>
+     </Labels>
+        </View>
+  </View>
+  <Header>
+    Document
+  </Header>
+  <View style="height: 600px; overflow: auto;">
+    <Text name="text" value="$text"/>
+  </View>
+</View>
+```
+
+**Related tags:**
+
+* [View](/tags/view.html)
+* [Text](/tags/text.html)
+* [Header](/tags/header.html)
+* [Labels](/tags/labels.html)
+
+## Check that TextArea input is valid JSON 
+
+This script parses the contexts of a TextArea field to check for valid JSON. If the JSON is invalid, it shows an error and prevents the annotation from being saved.
```

Comment:
It might be worth clarifying that compared with the obscenity example, this implementation hard-blocks the user from submitting their annotation.